packages/components/popover/popover.component.ts

```diff
@@ -58,10 +59,31 @@ export class KbqPopoverComponent extends KbqPopUp {
 
     isTrapFocus: boolean = false;
 
+    @ViewChild('popoverContent') popoverContent: ElementRef<HTMLDivElement>;
+
+    isContentTopOverflow: boolean = false;
+    isContentBottomOverflow: boolean = false;
+
     constructor(changeDetectorRef: ChangeDetectorRef) {
         super(changeDetectorRef);
     }
 
+    checkContentOverflow(): void {
+        const nativeElement = this.popoverContent.nativeElement;
+
+        if (!nativeElement) {
+            return;
+        }
+
+        const scrollTop: number = nativeElement.scrollTop;
+        const offsetHeight: number = nativeElement.offsetHeight;
+        const scrollHeight: number = nativeElement.scrollHeight;
+
+        this.isContentTopOverflow = scrollTop > 0;
+
+        this.isContentBottomOverflow = (((scrollTop as number) + offsetHeight) as number) < scrollHeight;
```

Comment:
непонятно зачем здесь столько скобок и привидение к number
```suggestion
        this.isContentBottomOverflow = scrollTop + offsetHeight < scrollHeight;
```